### PR TITLE
Added additional example

### PR DESCRIPTION
**Pull request by TheAlistairRoss**
Added an example to change an integer to a timespan

### CODE REVIEW

**Comment by PRMerger20** [comment]
@TheAlistairRoss : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by shsagir** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@shsagir: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: data-explorer/kusto/query/scalar-data-types/timespan.md [hunk 1]
```diff
@@ -49,4 +49,12 @@ print
     result1 = 1d / 1s,
     result2 = time(1d) / time(1s),
     result3 = 24 * 60 * time(00:01:00) / time(1s)
-```
\ No newline at end of file
+```
+
+This example converts an integer to a timespan by multiplying it by the timespan unit:
```
Comment (shsagir) [inline data-explorer/kusto/query/scalar-data-types/timespan.md:54]:
```suggestion
This example converts the number of seconds in a day (represented by an integer value) to a timespan unit:
```

**Comment by PRMerger17** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by shsagir** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@shsagir: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by orspod** [comment]
#sign-off

**Pull request merged**